packages/terra-form-select/src/shared/_Option.jsx

```diff
@@ -121,7 +121,7 @@ const Option = ({
   let label = display;
   // Allows VO to announce index of items
   if (SharedUtil.isMac() && index && totalOptions) {
-    label = `${display} (${index} ${ofText} ${totalOptions})`;
+    label = `${display}`;
```

Comment:
I think we don't need this `if()` block and variable label to update `itemLabel` with latest changes. instead we can directly use display as an value for `itemLabel`

```js
const itemLabel = isSelected || index === 1 ? `${expandedStateText} ${display}` : display;
```